core/jazz_acl/index.js

```diff
@@ -69,6 +73,12 @@ async function processACLRequest(event, config) {
       }
 
       await exportable.processScmPermissions(config, serviceId, policies, 'add');
+
+      logger.info("vault is enabled. global.globalConfig.VAULT.IS_ENALED : " + global.globalConfig.VAULT.IS_ENABLED)
+      if (global.globalConfig.VAULT.IS_ENABLED) {
```

Comment:
Do we need a check if `global.globalConfig.VAULT` is null? Make sure this doesn't fail if T-Vault extension is not installed.

Reply:
We don't need the check as it is coming from the config file. Also  it will have a false value if vault is not enabled